basicpermutationgroup.py

```diff
@@ -165,6 +165,50 @@ def stabilizer(generators: [Permutation], element: int) -> [Permutation]:
     return reduce(schreier_generators(generators, element), 0)
 
 
+def member_of(f: Permutation, H: [Permutation]) -> bool:
+    """
+    Returns whether the given permutation is generated by the generating set of permutations
+
+    :param Permutation f: permutation to check if it is in the generating set of permutations
+    :param [Permutation] H: generating set of permutations
+    :return bool: `True` if the permutation is generated by the generating set, `False` otherwise
+    """
+    # Base case: trivial permutation (or non trivial permutation in trivial generator set)
+    if len(H) == 0:
+        return f.istrivial()
+    if f.istrivial():
+        return True
```

Comment:
Is dit altijd waar? Tegenvoorbeeld(?): f=(), H=(0->1, 1->0) dan is f toch geen member van H?

Reply:
Ja dit zou altijd waar moeten zijn